packages/ui-extensions/src/surfaces/checkout/components/Heading/Heading.ts

```diff
@@ -2,7 +2,7 @@ import {createRemoteComponent} from '@remote-ui/core';
 
 import type {InlineAlignment, AccessibilityRole} from '../shared';
 
-type Level = 1 | 2 | 3;
+type Level = 1 | 2 | 3 | 4;
```

Comment:
@carolopolo tagging you here just to make sure since I'm not sure that we want to release this before we allow customizing this via the branding API? I'll defer to you.